.config-schema.json

```diff
@@ -435,24 +435,70 @@
             "type": "object",
             "properties": {
                 "enabled": {
-                    "description": "enable throttling when request's number of dispatches is high",
+                    "description": "DEPRECATED: Use OPENFGA_CHECK_DISPATCH_THROTTLING_ENABLED instead. enable throttling when request's number of dispatches is high",
                     "type": "bool",
                     "default": "false",
                     "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_ENABLED"
                 },
                 "frequency": {
-                    "description": "the frequency period that the deprioritized throttling queue is evaluated. A higher value will result in more aggressive throttling",
+                    "description": "DEPRECATED: Use OPENFGA_CHECK_DISPATCH_THROTTLING_FREQUENCY instead. the frequency period that the deprioritized throttling queue is evaluated. A higher value will result in more aggressive throttling",
                     "type": "duration",
                     "default": "10µs",
                     "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_FREQUENCY"
                 },
                 "threshold": {
-                    "description": "define the number of recursive operations to occur before getting throttled",
+                    "description": "DEPRECATED: Use OPENFGA_CHECK_DISPATCH_THROTTLING_THRESHOLD instead. define the number of recursive operations to occur before getting throttled",
                     "type": "integer",
                     "default": "100",
                     "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_THRESHOLD"
                 }
             }
+        },
+        "checkDispatchThrottling": {
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "enable throttling when check request's number of dispatches is high",
+                    "type": "bool",
+                    "default": "false",
+                    "x-env-variable": "OPENFGA_CHECK_DISPATCH_THROTTLING_ENABLED"
+                },
+                "frequency": {
+                    "description": "the frequency period that the deprioritized throttling queue is evaluated for a check request. A higher value will result in more aggressive throttling",
```

Comment:
actually lower value will result in more aggressive throttling (i.e., it will be evaluated more slowly)

Reply:
Seems like this is how we have documented in the previous implementation. I will change it all over?